src/internal/connector/graph/service.go

```diff
@@ -29,6 +29,7 @@ import (
 const (
 	logGraphRequestsEnvKey    = "LOG_GRAPH_REQUESTS"
 	log2xxGraphRequestsEnvKey = "LOG_2XX_GRAPH_REQUESTS"
+	log2xxGraphResponseEnvKey = "LOG_2XX_GRAPH_RESPONSES"
```

Comment:
I've added an option to log the response for 2xx entries as well. This can be a separate toggle that is enabled only when needed, but could come in handy for debugging failing of follow up requests like in the case of mail attachment fetch failure, but the request to get the mail succeeds.